packages/clerk-js/src/ui/components/Checkout/Checkout.tsx

```diff
@@ -35,7 +35,7 @@ const AuthenticatedRoutes = withCoreUserGuard((props: __experimental_CheckoutPro
     >
       <Drawer.Overlay />
       <Drawer.Content>
-        <Drawer.Header title='Checkout' />
+        <Drawer.Header titleSlot='Checkout' />
```

Comment:
Is `titleSlot` and `textSlot` something new? I don't see any usage of this pattern in `clerk-js`.

Reply:
Its prop names I am proposing we use. That support both passing a localization key or a ReactNode depending on the needs. Some of the commerce components have data that comes back like the plan name that I need to pass in to components but they also need to support localization keys.

Reply:
Updated to use `string | LocalizationKey` approach https://github.com/clerk/javascript/pull/5390/commits/aed0b08799cb5a0caa9cf8007982e65bffde98e5